build.sbt

```diff
@@ -1050,6 +1050,7 @@ lazy val `scio-google-cloud-platform` = project
       moduleFilter("org.apache.arrow", "arrow-vector"),
       moduleFilter("com.fasterxml.jackson.datatype", "jackson-datatype-jsr310")
     ).reduce(_ | _),
+    Test / javaOptions += "-Doverride.type.provider=com.spotify.scio.bigquery.validation.SampleOverrideTypeProvider",
```

Comment:
For context, BigQueryValidationTest was [failing](https://github.com/spotify/scio/actions/runs/13419010807/job/37486888149?pr=5611) because the system property `override.type.provider` used in tests was getting unset. Somehow this is happening in the test I added for `ConverterProviderTest` -- I set that test to `ignore` and the system property stayed set. Not sure why but let's just set it in the build?